BRLS/src/library/Main.java

```diff
@@ -7,265 +7,255 @@
+                        break;
+
+                    case "AI":
+                        addItem();
+                        break;
+
+                    case "LI":
+                        listItems();
+                        break;
+
+                    case "FI":
+                        fixItems();
+                        break;
+
+                    case "B":
+                        borrowItem();
+                        break;
+
+                    case "R":
+                        returnItem();
+                        break;
+
+                    case "L":
+                        listCurrentLoans();
+                        break;
+
+                    case "P":
+                        payFines();
+                        break;
+
+                    case "T":
+                        incrementDate();
+                        break;
+
+                    case "Q":
+                        finished = true;
+                        break;
+
+                    default:
+                        displayOutput("\nInvalid option\n");
+                        break;
+                }
+
+                Library.save();
+            }            
+        } catch (RuntimeException e) {
+            displayOutput(e);
+        }        
+        displayOutput("\nEnded\n");
+    }    
+
+    private static void payFines() {
+        pAY_fINE_cONTROL newPayFineControl = new pAY_fINE_cONTROL();
+        new PayFineUI(newPayFineControl).RuN();
+    }
+
+    private static void listCurrentLoans() {
+        displayOutput("");
+        for (Loan loan : library.listCurrentLoans()) {
+            displayOutput(loan + "\n");
+        }        
+    }
+
+    private static void listItems() {
+        displayOutput("");
+        for (Item book : library.listItems()) {
+            displayOutput(book + "\n");
+        }        
+    }
+
+    private static void listPatrons() {
+        displayOutput("");
+        for (Patron member : library.listPatrons()) {
+            displayOutput(member + "\n");
+        }        
+    }
+
+    private static void borrowItem() {
+        bORROW_IteM_cONTROL borrowItemControl = new bORROW_IteM_cONTROL();
+        new BorrowItemUI(borrowItemControl).RuN();
+    }
+
+    private static void returnItem() {
+        ReturnItemControl returnItemControl = new ReturnItemControl();
+        new ReturnItemUI(returnItemControl).run();
+    }
+
+    private static void fixItems() {
+        fIX_iTeM_cONTROL fixItemControl = new fIX_iTeM_cONTROL();
+        new FixItemUI(fixItemControl).RuN();
+    }
+
+    private static void incrementDate() {
+        try {
+
+            String prompt = "Enter number of days: ";
+            String display = getInput(prompt);
+            int days = Integer.valueOf(display);
+            calendar.incrementDate(days);
+            library.updateCurrentLoansStatus();
+            Date dates =  calendar.getDate();
+            String date = simpleDateFormat.format(dates);

```

Comment:
PR3 D13

According to Rule 2 of the ITC205 Code Style Guidelines, all variable names have to be useful - so I'd prefer if you took the same approach as you did with D12 and called the String variable stringSimpleDateFormat, and changed the Date variable to date.

Thanks :)